src/smite/smiteAPI.js

```diff
@@ -170,6 +170,20 @@ class Smite {
     })
   }
 
+  getGodLeaderBoard (godId, queue) {
+    let url = util.genUrl(this.smiteUrl, 'getgotleaderboard', this.devId, this.authKey, process.env[`SMITE_${this.platform}_SESSION`]) + '/' + godId +
```

Comment:
Other than that typo it looks great!  If you could, please also update the README.md with the new method.